app/main.py

```diff
@@ -2,5 +2,32 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    arguments = {}
```

Comment:
Maybe it’s worth naming the variable something else, considering that it will be our "cache"